code/datums/abilities/critter/voidhound.dm

```diff
@@ -0,0 +1,84 @@
+///////////////////////
+// Voidhound Abilities
+///////////////////////
+/datum/targetable/critter/voidhound/cloak
+	name = "Cloak"
+	desc = "Slip into the void and make yourself hard to discern for a short time"
+	icon_state = "clown_spider_bite"
+	cooldown = 40 SECONDS
+	targeted = 0
+
+	cast()
+		if (..())
+			return 1
+
+		animate(holder.owner, alpha=50, time=3 SECONDS)
+		boutput(holder.owner, "We slip into the shadows")
+		sleep(20 SECONDS)
```

Comment:
Can do! Should i make it so that players can reappear upon clicking the icon again, or make it so they cant attack/pounce and stay invisible at the same time?